extensions/blocks/podcast-player/components/playlist.js

```diff
@@ -8,7 +8,31 @@ import { __ } from '@wordpress/i18n';
 /**
  * Internal dependencies
  */
-import { STATE_ERROR } from '../constants';
+import * as episodeIcons from '../icons/episode-icons';
+import { STATE_ERROR, STATE_PLAYING } from '../constants';
+
+const TrackIcon = ( { isPlaying, isError, className } ) => {
+	let name;
+
+	if ( isError ) {
+		name = 'error';
+	} else if ( isPlaying ) {
+		name = 'playing';
+	}
+
+	const icon = episodeIcons[ name ];
+
+	if ( ! icon ) {
+		// Return empty element - we need it for layout purposes.
+		return <span className={ className } aria-hidden="true" />;
+	}
+
+	return (
+		<span className={ `${ className } ${ className }--${ name }` } aria-hidden="false">
```

Comment:
We likely don't need the `aria-hidden="false"` here.

Reply:
Removed in https://github.com/Automattic/jetpack/pull/15101/commits/a1dc70a4ad681d7c36639a38a84f06c1cf463dd2